scripts/test.sh

```diff
@@ -290,7 +307,8 @@ case "$TARGET" in
   popd
 
   pushd tests/react-test-app
-  npm install
+  npm install --no-save
```

Comment:
Why is `--no-save` needed here?

Reply:
Running the tests shouldn't update the lockfiles and without `--no-save` it sometimes will. Previously running test.sh would sometimes leave you with uncommitted changes that you had to manually discard.